src/main/resources/rs117/hd/lighting/lights.json

```diff
@@ -26521,8 +26521,8 @@
     "duration": 0.0,
     "range": 20.0,
     "objectIds": [
-      "FLAMES",
-      732
+      "FLAMES"
+      //732
```

Comment:
We're using a standard json parser that isn't expecting a comment here, you'll want to remove `732` outright.

```suggestion
```

Reply:
Comments are actually fine thanks to `setLenient()`, but we might as well remove this if it's not meant to be kept.

Reply:
I suppose maybe it's there to be reinstated whenever it's possible to restrict lights to a specific area? If so, I'd add a note about that on the commented line.

Reply:
Yeah it can come back later but i'l just remove it outright. The PR history will show ID or we can just go and check it again